trojsten/rules/test.py

```diff
@@ -1167,7 +1167,7 @@ def test_puzzle_hunt_participations(self):
         )
 
     def test_coefficient_all(self):
-        # Coefficient = 15: successful semesters = 2, other camps = 2,
+        # Coefficient = 16: successful semesters = 6, other camps = 2,
         # puzzlehunt participations = 3
```

Comment:
```suggestion
        # Coefficient = 16: successful semesters = 2, Susi camps = 1,
        # other camps = 2, puzzlehunt participations = 3
```
Podľa mňa tie čísla v komentári nesedia a mali by byť takto

Reply:
Hm to bol asi missclick, malo to byť 3, lebo ja som rátal successful semester dohromady aj body za ulohy aj susi sustredenia. Mozeme to updatenut na toto.